TheDeafOne/ScheduLite
# Cf/refactor styling

To test errors and warnings: 

for warnings:
- Add two of the same course in different sections to see warnings
- add more than 18 credits

for errors:
- add course overlap
